packages/orchestrator/lib/routes/v1/recurring/postRecurringRun.ts

```diff
@@ -0,0 +1,46 @@
+import { z } from 'zod';
+import type { Scheduler } from '@nangohq/scheduler';
+import type { ApiError, Endpoint } from '@nangohq/types';
+import type { EndpointRequest, EndpointResponse, RouteHandler, Route } from '@nangohq/utils';
+import { validateRequest } from '@nangohq/utils';
+
+const path = '/v1/recurring/run';
+const method = 'POST';
+
+export type PostRecurringRun = Endpoint<{
+    Method: typeof method;
+    Path: typeof path;
+    Body: {
+        scheduleName: string;
+    };
+    Error: ApiError<'recurring_run_failed'>;
+    Success: { scheduleId: string };
+}>;
+
+const validate = validateRequest<PostRecurringRun>({
+    parseBody: (data: any) => {
+        return z.object({ scheduleName: z.string().min(1) }).parse(data);
```

Comment:
I just noticed you are not using `strict()` in your zod objects validation. Would be great to add it, extra params can have insidious consequences when you spread object or else